docs/features/garbage_collector.md

```
@@ -0,0 +1,91 @@
+    .withPortBinding(5432, assignRandomHostPort: true)
+    .buildAsync()
+
+try await container.start()
+defer { Task { try? await container.stop(timeout: 10) } }
+
+// Test logic...
+```
+
+!!! tip
+
+    Use `try?` when stopping containers in cleanup code to avoid masking the original test failure with a cleanup error.
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)